### PR TITLE
revert: "fix: Fixes split panel bouncing on the edge between side/bott…

### PR DESCRIPTION
**Pull request by georgylobko**
…om when scrollbar appears (#2794)"

This reverts commit ca1cd898

### Description

Related links, issue #, if available: n/a

### How has this been tested?

<details>
   <summary>Review checklist</summary>

_The following items are to be evaluated by the author(s) and the reviewer(s)._

#### Correctness

- _Changes include appropriate documentation updates._
- _Changes are backward-compatible if not indicated, see [`CONTRIBUTING.md`](https://github.com/cloudscape-design/components/blob/main/CONTRIBUTING.md#public-apis)._
- _Changes do not include unsupported browser features, see [`CONTRIBUTING.md`](https://github.com/cloudscape-design/components/blob/main/CONTRIBUTING.md#browsers-support)._
- _Changes were manually tested for accessibility, see [accessibility guidelines](https://cloudscape.design/foundation/core-principles/accessibility/)._

#### Security

- _If the code handles URLs: all URLs are validated through [the `checkSafeUrl` function](https://github.com/cloudscape-design/components/blob/main/src/internal/utils/check-safe-url.ts)._

#### Testing

- _Changes are covered with new/existing unit tests?_
- _Changes are covered with new/existing integration tests?_
</details>

---

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: src/app-layout/__integ__/app-layout-drawers.test.ts [hunk 1]
```diff
@@ -209,7 +209,7 @@ describe.each(['classic', 'refresh', 'refresh-toolbar'] as const)('%s', theme =>
       )
     );
 
-    test(
+    test.skip(
```
Comment (jperals) [inline src/app-layout/__integ__/app-layout-drawers.test.ts:212]: Why is this test skipped?

I don't see it being unskipped in https://github.com/cloudscape-design/components/pull/2794/files

Comment (georgylobko) [reply src/app-layout/__integ__/app-layout-drawers.test.ts:212]: This test was written afterwards and relied on the PR's behavior. I spotted that it's failing and decided to skip it to speed up the process.

Comment (jperals) [reply src/app-layout/__integ__/app-layout-drawers.test.ts:212]: I think the viewport width defined in line 215 might need to be 1430, at least for Classic. Can you check that?

Comment (georgylobko) [reply src/app-layout/__integ__/app-layout-drawers.test.ts:212]: Tried. It does not change anything

**Review by jperals — APPROVED** [review]

**Review by jperals — APPROVED** [review]

**Pull request merged**